src/main/java/org/fundacionjala/at15/katas/langtonant/adrian/Board.java

```diff
@@ -0,0 +1,46 @@
+package org.fundacionjala.at15.katas.langtonant.adrian;
+
+public class Board {
+    private int width;
+    private int height;
+    private int[][] grid;
+
+    public Board(int width, int height) {
+        this.width = width;
+        this.height = height;
+        grid = new int[height][width];
+        for (int index = 0; index < grid.length; index++) {
+            for (int jndex = 0; jndex < grid[index].length; jndex++) {
+                grid[index][jndex] = 0;
+            }
+        }
+    }
+    public void print() {
+        for (int index = 0; index < grid.length; index++) {
+            for (int jndex = 0; jndex < grid[index].length; jndex++) {
+                System.out.print(grid[index][jndex] + " ");
+            }
+            System.out.println();
+        }
+    }
+
+    public int[][] getTablero() {
```

Comment:
* `getGrid()`

Reply:
No problem, I changed all the Spanish words